### PR TITLE
Change 'Kedro Viz' to 'Kedro Viz Run'

### PR DESCRIPTION
**Pull request by MehdiNV**
## Description
Resolves [1616](https://github.com/orgs/kedro-org/projects/1/views/4?pane=issue&itemId=43121094) and acts as a follow-up to [PR 1671](https://github.com/kedro-org/kedro-viz/pull/1671). 

This PR updates references of 'kedro viz' to be the new correct 'kedro viz run' instead.

## Development notes
Adjusts docs, simple textual changes 

## QA notes
Can be tested normally via `npm test`, the e2e tests, and directly using the command group in the CLI; should have no bearing on existing tests though, as changes are solely docs-specific

### CODE REVIEW

**Review by merelcht — APPROVED** [review]
LGTM! 👍

File: docs/source/extend_kedro/common_use_cases.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ def notebook_run(...):
     == ADD YOUR CUSTOM NOTEBOOK COMMAND CODE HERE ==
 ```
 
-To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and therefore must be a standalone plugin.
+To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz run` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and hence must be a standalone plugin.
```
Comment (stichbury) [inline docs/source/extend_kedro/common_use_cases.md:34]:
```suggestion
To inject additional CLI commands intended to be reused across projects, see [our plugin system](./plugins.md). An example of one such command is the `kedro viz run` command introduced by the [Kedro-Viz plugin](https://github.com/kedro-org/kedro-viz). This command is intended to work on every Kedro project and it must be a standalone plugin.
```

Comment (stichbury) [reply docs/source/extend_kedro/common_use_cases.md:34]: It's not that clear from reading this -- I'm left thinking "how is a command a standalone plugin"? So I've made an edit to try to clarify slightly but would appreciate some insight from @merelcht or @astrojuanlu or @noklam as to whether this now reads correctly.

**Review by stichbury — APPROVED** [review]
This looks good to me. I'm not seeing the same updates in the viz docs though -- does this mean that either (a) they need to be done or (b) they'll be dropped in only when they apply and currently `kedro viz` is the command to use and not `kedro viz run`.

**Comment by astrojuanlu** [comment]
Docs failure was transient:

```
WARNING: mermaid code 'flowchart TD\n  A[Start] --> B{Do you prefer developing your projects in notebooks?}\n  B -->|Yes| C[Use a Databricks workspace to develop a Kedro project]\n  B -->|No| D{Are you a beginner with Kedro?}\n  D -->|Yes| E[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  D -->|No| F{Do you have advanced project requirements<br>e.g. CI/CD, scheduling, production-ready, complex pipelines, etc.?}\n  F -->|Yes| G{Is rapid development needed for your project needs?}\n  F -->|No| H[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  G -->|Yes| I[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  G -->|No| J[Use a Databricks job to deploy a Kedro project]': Mermaid exited with error:
[stderr]
b'\nTimeoutError: Timed out after 30000 ms while waiting for the WS endpoint URL to appear in stdout!\n    at ChromeLauncher.launch (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/node/ProductLauncher.js:119:23)\n    at async run (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/src/index.js:404:19)\n    at async cli (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/src/index.js:184:3)\n\n'
[stdout]
b''
``` 

See https://github.com/kedro-org/kedro/issues/3395

File: docs/source/extend_kedro/common_use_cases.md (hunk 1 above)
Comment (astrojuanlu) [reply docs/source/extend_kedro/common_use_cases.md:34]: Hmmmm good point - the original phrasing is not clear to me either.

> This command is intended to work on every Kedro project and hence must be a standalone plugin.

Regardless of @stichbury edit, what does this mean?

**Comment by MehdiNV** [comment]
> Docs failure was transient:
> 
> ```
> WARNING: mermaid code 'flowchart TD\n  A[Start] --> B{Do you prefer developing your projects in notebooks?}\n  B -->|Yes| C[Use a Databricks workspace to develop a Kedro project]\n  B -->|No| D{Are you a beginner with Kedro?}\n  D -->|Yes| E[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  D -->|No| F{Do you have advanced project requirements<br>e.g. CI/CD, scheduling, production-ready, complex pipelines, etc.?}\n  F -->|Yes| G{Is rapid development needed for your project needs?}\n  F -->|No| H[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  G -->|Yes| I[Use an IDE, dbx and Databricks Repos to develop a Kedro project]\n  G -->|No| J[Use a Databricks job to deploy a Kedro project]': Mermaid exited with error:
> [stderr]
> b'\nTimeoutError: Timed out after 30000 ms while waiting for the WS endpoint URL to appear in stdout!\n    at ChromeLauncher.launch (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/node_modules/puppeteer-core/lib/esm/puppeteer/node/ProductLauncher.js:119:23)\n    at async run (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/src/index.js:404:19)\n    at async cli (file:///home/docs/.asdf/installs/nodejs/19.0.1/lib/node_modules/@mermaid-js/mermaid-cli/src/index.js:184:3)\n\n'
> [stdout]
> b''
> ```
> 
> See #3395

Ah makes a lot of sense; was completely lost by what this error was

**Comment by MehdiNV** [comment]
> This looks good to me. I'm not seeing the same updates in the viz docs though -- does this mean that either (a) they need to be done or (b) they'll be dropped in only when they apply and currently `kedro viz` is the command to use and not `kedro viz run`.

It'll be A yeah, they'll need to be done - I'll apply this over there as well ASAP

**Pull request merged**